arcgis-ios-sdk-samples/Maps/Change basemap/SwitchBasemapViewController.swift

```diff
@@ -16,40 +16,37 @@ import UIKit
 import ArcGIS
 
 class SwitchBasemapViewController: UIViewController {
-    @IBOutlet private weak var mapView: AGSMapView!
+    @IBOutlet private weak var mapView: AGSMapView! {
+        didSet {
+            // Initialize the map with the first basemap.
+            mapView.map = AGSMap(basemap: basemapInfoArray.first!.basemap)
+        }
+    }
     
     /// The basemap options plus labels.
     private let basemapInfoArray: [(basemap: AGSBasemap, label: String)] = [
-        (.darkGrayCanvasVector(), "Dark Gray Canvas (Vector)"),
-        (.imagery(), "Imagery (Raster)"),
-        (.imageryWithLabels(), "Imagery w/ Labels (Raster)"),
-        (.imageryWithLabelsVector(), "Imagery w/ Labels (Vector)"),
-        (.lightGrayCanvas(), "Light Gray Canvas (Raster)"),
-        (.lightGrayCanvasVector(), "Light Gray Canvas (Vector)"),
-        (.nationalGeographic(), "National Geograhpic (Raster)"),
-        (.navigationVector(), "Navigation (Vector)"),
-        (.oceans(), "Oceans (Raster)"),
-        (.openStreetMap(), "OpenStreetMap (Raster)"),
-        (.streets(), "Streets (Raster)"),
-        (.streetsVector(), "Streets (Vector)"),
-        (.streetsNightVector(), "Streets Night (Vector)"),
-        (.streetsWithReliefVector(), "Streets w/ Relief (Vector)"),
-        (.terrainWithLabels(), "Terrain w/ Labels (Raster)"),
-        (.terrainWithLabelsVector(), "Terrain w/ Labels (Vector)"),
-        (.topographic(), "Topographic (Raster)"),
-        (.topographicVector(), "Topographic (Vector)")
+        (.init(style: .arcGISDarkGrayBase), "Dark Gray Canvas (Vector)"),
+        (.init(style: .arcGISImageryStandard), "Imagery (Raster)"),
+        (.init(style: .arcGISImagery), "Imagery w/ Labels (Raster)"),
+        (.init(style: .arcGISLightGrayBase), "Light Gray Canvas (Raster)"),
+        (.init(style: .arcGISNavigation), "Navigation (Vector)"),
+        (.init(style: .arcGISNavigationNight), "Navigation Night (Vector)"),
+        (.init(style: .arcGISOceans), "Oceans (Raster)"),
+        (.init(style: .osmStreets), "OpenStreetMap (Raster)"),
+        (.init(style: .arcGISStreets), "Streets (Raster)"),
+        (.init(style: .arcGISStreetsNight), "Streets Night (Vector)"),
+        (.init(style: .osmStreetsRelief), "Streets w/ Relief (Vector)"),
+        (.init(style: .arcGISTerrain), "Terrain w/ Labels (Raster)"),
+        (.init(style: .arcGISTerrainDetail), "Terrain w/ Detail (Vector)"),
```

Comment:
It seems this style renders without terrain.

Reply:
I was also unsure about this style, `.osmStreetsRelief`, and `.arcGISTopographic`. They are most likely server issues and I assume that would be fixed by the release so I kept them there.

Reply:
`arcGISTerrainDetail` is documented as "A vector basemap with minimal map content like populated place names, admin and water labels with boundary lines as the reference layer. Designed to be overlaid on `AGSBasemapStyleArcGISTerrainBase` and hillshade." Therefore I don't think it is a server issue.

Reply:
@philium I see. I removed it from the sample. I also corrected the names of the basemaps. Please give this a review, thanks!

Reply:
Thanks! Since there's no longer vector/non-vector counterparts, perhaps we could just get rid of that distinction in the basemap names? That information is available in the documentation if desired.

Reply:
Good point! I got rid of them. Ready for your review!